src/components/views/dialogs/SetEmailDialog.js

```diff
@@ -106,7 +106,7 @@ export default React.createClass({
                 const QuestionDialog = sdk.getComponent("dialogs.QuestionDialog");
                 const message = _t("Unable to verify email address.") + " " +
                     _t("Please check your email and click on the link it contains. Once this is done, click continue.");
-                Modal.createDialog(QuestionDialog, {
+                Modal.createTrackedDialog('Verification Pending', 'M_THREEPID_AUTH_FAILED', QuestionDialog, {
```

Comment:
`M_THREEPID_AUTH_FAILED` looks like it came from somewhere special. I'd go for something matching the others like plain english